rpmalloc/rpmalloc.h

```diff
@@ -164,6 +164,10 @@ typedef struct rpmalloc_config_t {
 	//  passed is the number of bytes that was requested in the map call. Only used if
 	//  the default system memory map function is used (memory_map callback is not set).
 	int (*map_fail_callback)(size_t size);
+	//! Respectively allocated pages and huge allocated pages names for systems
```

Comment:
Minor personal taste - put the new config variables at the end of the structure (instead of the unused var that was cleaned out)

Reply:
alright you re the chief :-)